### DISCUSSION TITLE
https://dokdo.page/

### DISCUSSION THREAD

**Discussion by giscus** [sdk-xyz/blog-comments#10] [General]
# https://dokdo.page/

https://dokdo.page/

**Comment by sdk-kr** [comment]
![3669208aa2c020](https://github.com/sdk-xyz/blog-comments/assets/16316626/b5344ccb-086d-402e-9f0c-6bb35b1c3f9a)
![1c776eb24a8c16](https://github.com/sdk-xyz/blog-comments/assets/16316626/b3b3a1af-212c-423f-ad25-f03cedf8e4fd)

"日本が作成した地図にも、独島（日本の古い地名としての松島）は韓国の土地とされています。"
(English): "Even on maps created by Japan, Dokdo (formerly known in Japanese as Matsushima) is recognized as Korean territory."
Korea : 일본이 만든 지도에도 독도(일본 고 지명 송도)는 한국땅이라고 되어 있어요

**Comment by tkfkdgkqslek2** [comment]
https://damoang.net/data/editor/9f5f5-6635f2d6940be-29cf26f0d5475cd94323185be573853a55ce49da.png

**Reply by christlkh** [reply]
"Dokdo es territorio de Corea del Sur."

**Comment by kjhnew** [comment]
Dokdo is the territory of the Republic of Korea !!!

**Comment by christlkh** [comment]
"Dokdo es territorio de Corea del Sur."

**Comment by brandonKr** [comment]
Dokdo is the territory of South Korea
독도는 한국 땅 이야!

**Comment by feslento** [comment]
독도는 대한민국 영토입니다

Dokdo is the territory of the Republic of Korea

**Comment by redpakcha** [comment]
대마도가 원래 한국땅이었다!!
대동여지도에도 한국땅이었고, 미군들 지도에도, 다른 여러 역사지도에도 한국 영토로 나와있다!
대마도를 되찾자!!!

**Comment by redpakcha** [comment]
대마도가 대한민국 땅이라는 역사적 근거는 충분하다. 세종2년(1420년) 대마도를 정벌하고 500여년간 우리땅으로 복속됐다는 기록이 있다. 심지어 1952년 도요토미 히데요시가 조선침략을 위해 만든 팔도전도는 대마도를 조선영토로 표시하고 공격대상으로 적시했다.
